common/lib/xmodule/xmodule/tests/test_library_content.py

```diff
@@ -348,6 +350,52 @@ def _change_count_and_refresh_children(self, count):
         assert len(selected) == count
         return selected
 
+    def mock_reset_problem(self, _data):
+        """
+        Mocks ProblemBlock's reset problem
+        """
+        return {
+            'success': True
+        }
+
+    def test_reset_selected_children(self):
+        """
+        Tests that a content block is reset when
+        show_reset_button is True
+        """
+        self.lc_block.show_reset_button = True
+        self.lc_block.max_count = 2
+        self._create_capa_problems()
+        self.lc_block.capa_type = "optionresponse"
+        self.lc_block.refresh_children()
+        self.lc_block = self.store.get_item(self.lc_block.location)
+        self._bind_course_module(self.lc_block)
+
+        with patch.object(ProblemBlock, 'reset_problem') as reset_problem:
+            reset_problem.side_effect = self.mock_reset_problem
+            response = self.lc_block.reset_selected_children(None)
+
+        assert response.status == '200 OK'
+
+    def test_reset_selected_children_closed(self):
+        """
+        Tests that a content block is not reset when
+        show_reset_button is False
+        """
+        self.lc_block.show_reset_button = False
+        self.lc_block.max_count = 2
+        self._create_capa_problems()
+        self.lc_block.capa_type = "optionresponse"
+        self.lc_block.refresh_children()
+        self.lc_block = self.store.get_item(self.lc_block.location)
+        self._bind_course_module(self.lc_block)
+
+        with patch.object(ProblemBlock, 'reset_problem') as reset_problem:
+            reset_problem.side_effect = self.mock_reset_problem
+        response = self.lc_block.reset_selected_children(None)
+
+        assert response.status == '400 Bad Request'
```

Comment:
```suggestion
        assert response.status_code == status.HTTP_400_BAD_REQUEST
```

We should use constants instead of plain strings.

Reply:
@Agrendalath  The status of the `Response` object used by the `reset_selected_children` method  returns `400 Bad Request` meanwhile `status.HTTP_400_BAD_REQUEST` returns just `400`